+++
title = "Next Generation Sequencing"
lastmod = 2021-06-16T12:17:07-07:00
draft = false
+++

## NGS and [Machine Learning]({{<relref "machine_learning.md" >}}) {#ngs-and-machine-learning--machine-learning-dot-md}

-   The error rates of NGS reads range from ~0.1–10%. <sup id="95732a8d4c6d4164edf7cd9544e09db8"><a href="#poplin" title="Poplin, Chang, Alexander, Schwartz, Colthurst, Ku, Newburger, Dijamco, Nguyen, Afshar \&amp; others, A universal SNP and small-indel variant caller using deep neural networks, {Nature biotechnology}, v(10), 983--987 (2018).">poplin</a></sup>
-   The previous best variant calling algorithms were highly specialized to the Illumina platform, and are thus
-   In 2018, engineers at Google demonstrated a convolutional neural network—a prominent class of neural network used in computer vision—named DeepVariant that could identify SNP and indel mutations more than 50% more accurately than the next best algorithm. <sup id="95732a8d4c6d4164edf7cd9544e09db8"><a href="#poplin" title="Poplin, Chang, Alexander, Schwartz, Colthurst, Ku, Newburger, Dijamco, Nguyen, Afshar \&amp; others, A universal SNP and small-indel variant caller using deep neural networks, {Nature biotechnology}, v(10), 983--987 (2018).">poplin</a></sup>

    # Bibliography
    <a id="poplin"></a>[poplin] Poplin, Chang, Alexander, Schwartz, Colthurst, Ku, Newburger, Dijamco, Nguyen, Afshar & others, A universal SNP and small-indel variant caller using deep neural networks, <i>Nature biotechnology</i>, <b>36(10)</b>, 983-987 (2018). [↩](#95732a8d4c6d4164edf7cd9544e09db8)